apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -40,6 +40,8 @@ const MultiAnswerStatus = {
   INCORRECT: 'incorrect'
 };
 
+const ANSWER_LETTERS = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'];
```

Comment:
Do these need to be translated or are these always the values we use for assessments?

Reply:
:man_shrugging: Let's ask the curriculum team.

Reply:
I'll leave this as a TODO and follow up with Poorva.